timescaledb/how-to-guides/backup-and-restore/pg-dump-and-restore.md

```diff
@@ -29,6 +29,11 @@ example, to backup a database named `tsdb`:
 ```bash
 pg_dump -Fc -f tsdb.bak tsdb
 ```
+To backup a database named `tsdb` hosted on a remote server:
+
+```bash
+pg_dump -h <REMOTE_HOST> -p -U -f tsdb.bak tsdb
```

Comment:
I tested a few variations using a Timescale Cloud database. This one doesn't seem to work. Maybe something like this should be better (tested, it works):
```
pg_dump -h xxx.xxx.tsdb.cloud.timescale.com -p 55555 -U tsdbadmin -Fc -f tsdb.bak tsdb
```
This one shows a Timescale Cloud example, but you can change the host to <REMOTE_HOST> if you want!

Reply:
done!!!